dropwizard-client/src/main/java/io/dropwizard/client/ssl/TlsConfiguration.java

```diff
@@ -38,6 +40,10 @@
     @Nullable
     private List<String> supportedCiphers = null;
 
+    @Nullable
+    @Size(min=1)
```

Comment:
Ensuring that if a value for certAlias is provided it's min length is 1. Maybe this is unnecessary.